tutorials/3d/standard_material_3d.rst

```diff
@@ -26,15 +27,19 @@ added in the *Material Override* property of the node, it will only be used by t
 It will also override the regular material property of the node and the material property of
 the mesh.
 
-Flags
------
+BaseMaterial 3D Settings
+------------------------
 
-Spatial materials have many flags determining the general usage of a material.
+StandardMaterial3D has many settings that determine the look of a material. All of these are
+under the BaseMaterial3D category
 
 .. image:: img/spatial_material1.png
 
-Transparent
-~~~~~~~~~~~
+ORM materials are almost exactly the same with one difference. Instead of separate settings
+and textures for occlusion, roughness, and metallic, there is a single ORM texture.
```

Comment:
```suggestion
and textures for occlusion, roughness, and metallic, there is a single ORM texture.
```

Maybe add a single sentence how ORM textures work (i.e. containing one map in each color channel, I assume?), and how they're commonly authored? May make sense for newer users to mention this briefly for context.